content/software-security/what-are-containers/index.md

```diff
@@ -0,0 +1,65 @@
+
+There are many factors to consider when choosing a container image for deploying your applications. At minimum, your container image will need to have the core packages and components required to support your programs. However, with thousands of options to choose from, some images will be better for your use case than others. Choosing a container image that offers security and reliability in addition to core functionality is important. Some images may contain multitudes of unneeded components and packages, thus unnecessarily contributing to your data egress and vulnerability counts.
+
+To learn more about choosing a container image that is right for your applications, check out our article on [selecting a base image](/software-security/selecting-a-base-image/).
+
+
+## Learn More
+
+As container technology grew in popularity, developers sought a way to standardize the format of images within the container industry. Thus, in 2015, the [Open Container Initiative (OCI)](https://opencontainers.org/) was founded to address these concerns by setting forth adoptable standards for image formats, runtimes, and distributions. Choosing OCI compliant images for your workloads can ensure the portability of your containers where OCI images are supported. To learn more about the OCI and their various image specifications, check out our article titled ["What is the Open Container Initiative?"](/open-source/oci/what-is-the-oci/).
+
+In this article, you learned about the basics of container technology, including the processes behind building container images tailored to your specific needs. You learned about how you can deploy containers from images using a container engine. To get started with using containers for the first time, check out our documentation on [Chainguard Images](/chainguard/chainguard-images/), our hardened, minimal images ideal for deploying secure containerized applications. If you want to build your own images, check out [Wolfi](/open-source/wolfi/), the Linux undistro ideal as a base of lightweight containers.
```

Comment:
```suggestion
In this article, you learned about the fundamentals of container technology, including the processes behind building container images tailored to your specific needs. You also learned about how you can deploy containers from images using a container engine.

To get started with using containers for the first time, check out our documentation on [Chainguard Images](/chainguard/chainguard-images/), our hardened, minimal images ideal for deploying secure containerized applications. If you want to build your own images, check out [Wolfi](/open-source/wolfi/), the Linux undistro ideal as a base of lightweight containers. Lastly, you may also be interested in learning about the Open Container Initiative which sets standards for image formats, runtimes, and distributions. We encourage you to check out our article titled ["What is the Open Container Initiative?"](/open-source/oci/what-is-the-oci/).
```

I think it's a little late in the article to introduce this much info about the OCI. This suggestion tries keeping that detail (because it is an interesting thing readers might want to dig into) while making its introduction less abrupt.

Reply:
This was based on my feedback, to git blame me :)

Reply:
I prefer this way of tying it in, thank you for the suggestion! :O